webpack.config.js

```diff
@@ -9,6 +9,10 @@ var paths = require('./scripts/paths.js');
 var englishStrings = require('./locales/en-US.json');
 var currencies = require('./data/currencies.js');
 var url = require('url');
+var Habitat = require('habitat');
```

Comment:
apparently we need this for `npm install` because of line 76 where we need to do `url.resolve(process.env.APPLICATION_URI...) and it is undefined.